### PR TITLE
ChongSoon_ basics_9_1

### PR DESCRIPTION
**Pull request by HelloChongSoon**
Please fill out the survey before submitting the pull request. Thanks!

🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀

How many hours did you spend on this assignment? _2_

Please fill in one error and/or error message you received while working on this assignment. _syntax error_

What part of the assignment did you spend the most time on? _logic thinking_

Comfort Level (1-5): _3_

Completeness Level (1-5): _3_

What did you think of this deliverable? _hard at first but kind of easy in hindsight_

Is there anything in this code that you feel pleased about? _able to put everything learnt so far into context_

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,4 +1,79 @@
...
+};
+
+var gameStart = function (userName, userGuess) {
+  // Rules: scissors beats paper, paper beats stone, and stone beats scissors. If both parties choose the same object, it's a draw.
+  var robotHands = generateRandomHands();
+  var gameMessage = "";
+  // outputs whether the user won, the program won, or it's a draw.
+  var userWin = `${userName} choose ${userGuess}, <br> Bot choose ${robotHands}, <br> You Won! <br><br> Total Wins: <br> ${userName} VS Bot <br> ${playerWinCounter}:${botWinCounter}`;
+  var userLose = `${userName} choose ${userGuess}, <br> Bot choose ${robotHands}, <br> You Lose! Hahaha <br><br> Total Wins: <br> ${userName} VS Bot <br> ${playerWinCounter}:${botWinCounter}`;
+  var draw = `${userName} choose ${userGuess}, <br> Bot choose ${robotHands}, <br> It's a draw! <br><br> Total Wins: <br> ${userName} VS Bot <br> ${playerWinCounter}:${botWinCounter}`;
```
Comment (eddiejpot) [inline script.js:32]: Great work separating your message outputs. might be good to put this in a separate function 
E.g. 

```
var outputMessage = function (messageType) {
    if (messageType == 'userWin') {
        return ...
    }
    if (messageType == 'userLose') {
        return ...
    }
    if (messageType == 'draw') {
        return ...
    }
} 

```

// use this when needed 
`outputMessage ('userWin')`

File: script.js [hunk 2]
```diff
@@ -1,4 +1,79 @@
...
+    return (gameMessage = userWin);
+  }
+  // user lose
+  if (
+    (robotHands == paper && userGuess == stone) ||
+    (robotHands == stone && userGuess == scissors) ||
+    (robotHands == scissors && userGuess == paper)
+  ) {
+    botWinCounter = botWinCounter + 1;
+    return (gameMessage = userLose);
```
Comment (eddiejpot) [inline script.js:53]: Great work using || && operators here !

**Review by eddiejpot — COMMENTED** [review]
Great work ! Looks like you understand global variables, functions and conditionals well!

File: script.js (hunk 1 above)
Comment (HelloChongSoon) [reply script.js:32]: Good idea!

**Pull request closed without merging**